mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -191,3 +194,19 @@ object English : LocalizedStrings {
   override val tournamentsFilterOnlyAdministrating = "Administrating".uppercase()
   override val tournamentsFilterBackContentDescription = "Back"
 }
+
+/**
+ * Converts a [Duration] to an English string by rounding it to the closest unit of time (seconds
+ * minimum).
+ */
+private fun Duration.toEnglishString(): String {
+  if (this >= 1.days) {
+    return "${this.inWholeDays} days"
+  } else if (this >= 1.hours) {
+    return "${this.inWholeHours} hours"
+  } else if (this >= 1.minutes) {
+    return "${this.inWholeMinutes} minutes"
+  }
+
+  return "${this.inWholeSeconds} seconds"
```

Comment:
Why the return "${this.inWholeSeconds} seconds" isn't in an else branch ? Do you have a particular reason to do that ?